webrtc.html

```diff
@@ -8857,6 +8859,58 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       [[!WEBRTC-STATS]].</p>
     </section>
     <section>
+      <h3>The stats selection algorithm</h3>
+      <p>
+	The <dfn>stats selection algorithm</dfn> is as follows:
+	<ul>
+          <li>
+            Let <var>result</var> be an empty RTCStatsReport.
+          </li>
+	  <li>If the selector is a MediaStreamTrack, add the following
+            objects to <var>result</var>, if they exist:
+	    <ul>
+	      <li>
+		The RTCMediaStreamTrackStats corresponding to the selector.
+	      </li>
+	      <li>
+		If the selector track is inbound, the RTCInboundRTPStreamStats
+                corresponding to the selector that has "remote=false", and the
+                RTCOutboundRTPStreamStats corresponding to the selector that has
+                "remote=true".
```

Comment:
It is my understanding that an inbound track has a corresponding RTCInboundRTPStreamStats (or multiple if receiving multicast is supported - can you clarify that it could be plural if so?), but never RTCOutboundTPStreamStats.

Is this comment incorrect or in what case would there be RTCOutboundRTPStreamStats corresponding to an inbound track?